.buildkite/pipelines/es_serverless/verify_es_serverless_image.yml

```diff
@@ -82,6 +82,22 @@ steps:
             - exit_status: '*'
               limit: 1
 
+      - command: .buildkite/scripts/steps/test/scout_uiam_tests.sh
```

Comment:
__note:__ currently we cannot run Scout tests with a custom config within the existing Scout pipeline, the recommended temporary approach is to create a dedicated pipeline.

That being said, soon we'll enable UIAM by default and drop our custom config and this custom pipeline.